test/integration/PetService.test.ts

```diff
@@ -0,0 +1,30 @@
+import { Container } from 'typedi';
+import { createConnection, useContainer, Connection } from 'typeorm';
+
+import { Pet } from '../../src/api/models/Pet';
+import { PetService } from './../../src/api/services/PetService';
+import { createDatabaseConnection, synchronizeDatabase, closeDatabase } from './utils/database';
+
+describe('PetService', () => {
+
+    let connection: Connection;
+    beforeAll(async () => connection = await createDatabaseConnection());
+    beforeEach(() => synchronizeDatabase(connection));
+    afterAll(() => closeDatabase(connection));
```

Comment:
Even though the `closeDatabase` function didn't exist before my review commits it has not thrown a error in TS.
Based on this fact any TS errors in integration tests, which don't fail the app hard, will not be detected by the developer.